index.html

```diff
@@ -107,20 +107,20 @@ <h1>About <br> MySelf</h1>
             <h3>LET'S  CONNECT</h3>
```

Comment:
- [ ] Please the container of the about section should be a section tag since it represents sectioning content.